Automattic/jetpack
# Connection: deactivate the Connection Manager UI

#### Changes proposed in this Pull Request:
* Remove activation of the Connection Manager UI from Jetpack and Backup plugins.
* Edit the mu-plugin to activate Connection Manager UI on the dev environments.

#### Jetpack product discussion
p1637178161039900-slack-C025MP1Q3HT

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Setup a JN site with both Jetpack and Backup plugins active on this branch.
2. Confirm that the "Tools -> Connection Manager" menu item **does not** show up.
3. Checkout the branch locally in the Monorepo, confirm that it **does** show up.
